test/erlcloud_ddb2_tests.erl

```diff
@@ -69,7 +69,19 @@ operation_test_() ->
       fun update_table_input_tests/1,
       fun update_table_output_tests/1,
       fun update_time_to_live_input_tests/1,
-      fun update_time_to_live_output_tests/1
+      fun update_time_to_live_output_tests/1,
+      fun list_backups_input_tests/1,
```

Comment:
Similar issue here. The module has an ordering for tests, and you're just tacking onto the end. please add these references (and the functions) in symmetric ordering to the erlcloud_ddb2 module.